certified-connectors/IN-D-Payables/README.md

```diff
@@ -0,0 +1,15 @@
+## IN-D Payables
```

Comment:
@gayathri35 Please follow the [default template provided for readme.md file](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/readme.md) and update it accordingly with required information.